### PR TITLE
RHPAM-1319 RHPAM-1319 Not possible to edit existing Deployment Descriptor configuration

### PR DESCRIPTION
**Pull request by bxf12315**
@cristianonicolai The PR related to https://github.com/kiegroup/jbpm-wb/pull/1181

### CODE REVIEW

**Comment by bxf12315** [comment]
@cristianonicolai  Please review it.

**Comment by bxf12315** [comment]
@tiagobento  Please review it.

**Comment by bxf12315** [comment]
@cristianonicolai @tiagobento I updated the PR, Please review it.

File: kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-client/src/main/java/org/kie/workbench/common/screens/library/client/settings/sections/persistence/properties/PropertiesItemView.java [hunk 1]
```diff
@@ -66,4 +76,14 @@ public void setName(final String name) {
...
+
+//    @Override
+//    public String getName() {
+//        return this.name.textContent;
+//    }
+//
+//    @Override
+//    public String getValue() {
+//        return this.value.textContent;
+//    }
```
Comment (tiagobento) [inline kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-client/src/main/java/org/kie/workbench/common/screens/library/client/settings/sections/persistence/properties/PropertiesItemView.java]: Commented code can be removed I suppose.

File: kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-client/src/main/java/org/kie/workbench/common/screens/library/client/settings/sections/validation/ValidationPresenter.java [hunk 2]
```diff
@@ -92,4 +92,5 @@ public ValidationListPresenter(final ManagedInstance<ValidationItemPresenter> it
             super(itemPresenters);
         }
     }
+
```
Comment (tiagobento) [inline kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-client/src/main/java/org/kie/workbench/common/screens/library/client/settings/sections/validation/ValidationPresenter.java]: Blank line

File: kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-client/src/main/java/org/kie/workbench/common/screens/library/client/settings/util/modal/doublevalue/AddDoubleValueModalView.java [hunk 3]
```diff
@@ -134,4 +134,21 @@ public void setNameLabel(final String label) {
     public void setValueLabel(final String label) {
         this.valueLabel.textContent = label;
     }
+
+    @Override
+    public HTMLElement getElement() {
+        // TODO Auto-generated method stub
+        return null;
+    }
```
Comment (tiagobento) [inline kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-client/src/main/java/org/kie/workbench/common/screens/library/client/settings/util/modal/doublevalue/AddDoubleValueModalView.java]: Isn't this breaking the screen?

File: kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-client/src/main/java/org/kie/workbench/common/screens/library/client/settings/sections/persistence/properties/PropertiesItemView.java (hunk 1 above)
Comment (bxf12315) [reply kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-client/src/main/java/org/kie/workbench/common/screens/library/client/settings/sections/persistence/properties/PropertiesItemView.java]: ok

File: kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-client/src/main/java/org/kie/workbench/common/screens/library/client/settings/sections/validation/ValidationPresenter.java (hunk 2 above)
Comment (bxf12315) [reply kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-client/src/main/java/org/kie/workbench/common/screens/library/client/settings/sections/validation/ValidationPresenter.java]: ok

File: kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-client/src/main/java/org/kie/workbench/common/screens/library/client/settings/util/modal/doublevalue/AddDoubleValueModalView.java (hunk 3 above)
Comment (bxf12315) [reply kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-client/src/main/java/org/kie/workbench/common/screens/library/client/settings/util/modal/doublevalue/AddDoubleValueModalView.java]: ok will remove it

**Comment by bxf12315** [comment]
@cristianonicolai old solution, so I close it.

**Pull request closed without merging**